pkg/config/containers.conf

```diff
@@ -381,6 +381,9 @@ default_sysctls = [
 #
 #machine_enabled = false
 
+# MachineImage is the image used when creating a podman-machine VM
```

Comment:
In the comment (here and in the other file below), the MixedCase bothers me. Rather than synchronize with the spelling of the actual option, I'd suggest just removing the duplication entirely: "The image used when ..."